x-pack/plugins/apm/server/lib/correlations/queries/field_stats/get_field_stats.test.ts

```diff
@@ -76,7 +69,7 @@ describe('field_stats', () => {
               },
             },
           },
-          sampler: { shard_size: 5000 },
+          sampler: { shard_size: 10000 },
```

Comment:
Can we use `SAMPLER_SHARD_SIZE` from `x-pack/plugins/apm/common/correlations/constants.ts`?